pyaedt/icepak.py

```diff
@@ -5052,23 +4973,12 @@ def assign_resistance(self, objects, boundary_name=None, total_power="0W", fluid
         boundary_name : str, optional
             The name of the boundary object that will be created. If not
             provided, a unique name will be generated. Default is ``None``.
-        total_power : str, float, or dict, optional
+        total_power : str, float, or dict or BoundaryDictionary, optional
             The total power transferred to the fluid through the resistance
             volume. It is specified as a string with value and unit, a float
-            where the default unit "W" will be used, or a dictionary for
-            transient assignment. The dictionary should contain two keys:
-            ``"Function"`` and ``"Values"``.
-
-            - For the ``"Function"`` key, options are ``"Exponential"``,
-            ``"Linear"``, ``"Piecewise Linear"``, ``"Power Law"``,
-            ``"Sinusoidal"``, and ``"Square Wave"``.
-            - For the ``"Values"`` key, provide a list of strings containing
-            the parameters required by the ``"Function"`` key selection. For
-            example, when ``"Linear"`` is set as the ``"Function"`` key, two
-            parameters are required: the value of the variable at t=0 and the
-            slope of the line. For the parameters required by each
-            ``"Function"`` key selection, see the Icepak documentation.
-
+            where the default unit "W" will be used.
+            Assign a transient condition using the result of a function with
+            the pattern `create_*_transient_assignment`.
```

Comment:
```suggestion
            where the default unit "W" is used.
            Assign a transient condition using the result of a function with
            the pattern ``create_*_transient_assignment``.
```
Avoid future tense. I did edit this one since I was adding the other edit.